README.md

```diff
@@ -151,15 +149,15 @@ exports.buildStyles = buildStyles;
 
 As of version 5, `gulp-sass` _does not include a default Sass compiler_, so you must install one (either `node-sass` or `sass`) along with `gulp-sass`.
 
-```
+```sh
 npm install sass gulp-sass --save-dev
 ```
 
 Then, you must explicitly set that compiler in your gulpfille. Instead of setting a `compiler` prop on the `gulp-sass` instance, you pass the compiler into a function call when instantiating `gulp-sass`.
 
 These changes look something like this:
 
-``` diff
+```diff
 - var sass = require('gulp-sass'));
 - var compiler = require('sass');
 - sass.compiler = compiler;
```

Comment:
const here too?

Reply:
this is a diff so I wasn't sure if I should change it...

Reply:
Keeping consistent is good. There'll be lots of JS folks who have never been exposed to `var` before.

Reply:
Not sure if it's valid, but I changed these `var`s to `const`.